smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/connectors/WorkerPoolRegistry.java

```diff
@@ -38,9 +37,6 @@
     @Inject
     private ExecutionHolder executionHolder;
 
-    @Inject
```

Comment:
What's the reason for not injecting the Config and using the Provider?

I thought that was a source of issues?

Reply:
When I had this code in Quarkus, it caused problems, I will try to roll back to Instance<Config>